training/train_grpo.py

```diff
@@ -0,0 +1,127 @@
+"""Train an LLM using GRPO over Reasoning Gym procedural dataset(s)."""
+
+from dataclasses import replace
+
+import hydra
+import ray
+from omegaconf import OmegaConf
+from trainers import RayGRPOTrainer
+from utils import ReasoningGymDataset, make_dataset
+
+import reasoning_gym
+import reasoning_gym.utils
+from reasoning_gym.coaching.curriculum_config import CurriculumAttributeConfig, CurriculumExperimentConfig
+from reasoning_gym.coaching.experiment import CurriculumExperiment
+from reasoning_gym.composite import CompositeDataset, DatasetSpec
+
+
+def prepare_datasets(config, tokenizer) -> tuple[ReasoningGymDataset, ReasoningGymDataset]:
+    """Prepare training and validation datasets."""
+    dataset_size = config.reasoning_gym.dataset_size
+    developer_prompt = reasoning_gym.utils.SYSTEM_PROMPTS["DeepSeekZero"]
```

Comment:
would love for this to be optionally exposed in config.reasoning_gym.developer_prompt